docs/images.md

```diff
@@ -112,9 +131,9 @@ Response:
     Content-Length: 122
 
     {"unix_socket": "\u0000/org/ovirt/imageio", "features": ["extents", "zero", "flush"],
-     "max_readers": 8, "max_writers": 8}
+     "max_readers": 8, "max_writers": 8, "format": "raw", "size": 53687091200}
```

Comment:
The docs discuss `virtual_size` but here we report `size`. I think reporting `virtaul_size` will be more clear. This is also the same term used in `qemu-img info`.